be/user.go

```diff
@@ -3,16 +3,43 @@ package be
 import (
 	"log"
 	"net/http"
+	"strconv"
 
 	"github.com/gin-gonic/gin"
 )
 
+type TeamColor int
+
+const (
+	Unknown = 0
+	Blue    = 1
+	Greed   = 2
+)
+
+func UserIdToTeam(id string) TeamColor {
+	if len(id) > 2 {
+		id = id[:len(id)-2]
+	}
```

Comment:
Remember that the id starts with 0x. The correct way would be
if len(id) > 2 && (id[1] == 'x' || id[1] == 'X') {
  id = id[2:min(18, len(a))]
} else {
  id = id[:min(16, len(a))]
}

Reply:
Checking. Odd, it worked. Verifying.

Reply:
Also, WTH we keep that 0x prefix?

Reply:
We just take it as is, it's generated not by us. So we don't make any changes in it

Reply:
![Screenshot 2023-11-18 at 20 52 38](https://github.com/cleanappio/cleanapp_back_end_v2/assets/20050859/35e0317a-3556-4cb3-8bfd-d07d2ff9e710)


I don't understand what is the idea of cutting the last char from the ID.

Reply:
I assumed that we never had (and should not have) such short IDs. I did not like the idea of using the last character, since it may be less randomised. Only thhat. Let's chat on Monday.